coalib/output/ConsoleInteraction.py

```diff
@@ -609,6 +609,51 @@ def print_actions(console_printer, section, actions, failed_actions):
     return get_action_info(section, actions[choice - 1], failed_actions)
 
 
+def apply_action_to_section(action,
+                            section,
+                            action_name,
+                            result,
+                            file_dict,
+                            file_diff_dict,
+                            console_printer,
+                            failed_actions):
+    """
+    Applies the given action.
+
+    :param action:          The action to the applied.
+    :param section:         Currently active section.
+    :param action_name:     The name of the action.
```

Comment:
But this would make the function less usable, so I think better to remove it :+1: